prow/plugins/updateconfig/updateconfig.go

```diff
@@ -174,10 +176,7 @@ func Update(fg FileGetter, kc corev1.ConfigMapInterface, name, namespace string,
 	return nil
 }
 
-// ConfigMapID is a name/namespace combination that identifies a config map
-type ConfigMapID struct {
-	Name, Namespace string
-}
+type ConfigMapID plugins.ConfigMapID
```

Comment:
Is this worth it? Can't we just use the type directly?

Reply:
Done.